services/apis/fence/fenceTasks.js

```diff
@@ -48,6 +48,20 @@ module.exports = {
     return response.data.version;
   },
 
+  /**
+   * Runs a fence command to rotate a client's ID and secret
+   * @param {string} clientName - client name
+   */
+  rotateClientCredentials(clientName, expiresInDays) {
+    let fenceCmd = `fence-create client-rotate --client ${clientName}`;
+    if (expiresInDays) {
+      fenceCmd = `${fenceCmd} --expires-in ${expiresInDays}`;
+    }
+    const res = bash.runCommand(fenceCmd, 'fence', takeLastLine);
+    const arr = res.replace(/[()']/g, '').split(',').map((val) => val.trim());
```

Comment:
possible to add an example of res in the docstrings for making it easy to understand the goal of this line for newcomers?